extensions/amp-story/1.0/progress-bar.js

```diff
@@ -382,6 +389,21 @@ export class ProgressBar {
     });
   }
 
+  /**
+   * Called from updateProgress method to set the `aria-label` attribute
+   * triggering assistive technology to inform user of the current page.
+   * @param {number} index zero-based index of pages in the story.
+   * @private
+   */
+  setAriaLabelByIndex_(index) {
+    this.mutator_.mutateElement(this.getRoot(), () => {
+      this.getRoot().setAttribute(
+        'aria-label',
+        `page ${index + 1} of ${this.segmentCount_}`
```

Comment:
Should this be translated into different languages?

Reply:
I think that is a good idea.  I can look for examples on how to reference files in `_locales`, then create a new object for the words. 

```json
"63": {"string": "page"},
"64": {"string": "of"}
```

Reply:
With the existing localization API, this is going to translate pretty poorly.  We'd need string interpolation/placeholders for this to localize well (i.e. translators should see "Page X of Y" instead of "page" and "of" with string concatenation).  It's not the case that it translates literally in every language.

I'm not entirely sure what the right short term fix is, though.  Open to ideas

Reply:
@gmajoulet and I had a chat and we think the best way forward is to add support for variable substitution in our localization strings. 

The idea would be to support something like `en: "Page {{index}} of {{count}}"`. @gmajoulet will create a GH issue with the details.

Reply:
No problem! I will find another issue to look at in the meantime :)

Reply:
Thank you Nick, we'll be working on this localization thing very actively so this can be unblocked!

Reply:
Hi @gmajoulet, I wanted to check-in on variable substitution with localization strings. I found another issue (32516) related to accessible names for control elements, but I think it will require language support like this issue does.

Reply:
We've been struggling to define an API that's powerful enough but doesn't require hundreds/thousands of lines added to the main JavaScript bundle (insanely high traffic). To be honest this project is no longer on our roadmap :( While it's def something we'd love to support, I don't think we'll be able to make any progress on it this year

Reply:
Ah ok, thanks for letting me know - I'll close this PR and see if [32516](https://github.com/ampproject/amphtml/issues/32516) can be addressed without the string localization.